### PR TITLE
Document that compilation target must be added with rustup

### PR DESCRIPTION
**Pull request by afajl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,13 @@ In `.cargo/config.toml`, pick the right compilation target for your board.
 +target = "thumbv7em-none-eabihf" # Cortex-M4F (with FPU)
 ```
 
+Add the target with rustup.
+
+``` console
+$ rustup target add thumbv7em-none-eabihf
+```
+
+
```
Comment (Urhengulas) [inline README.md:83]:
```suggestion
Add the target with `rustup`.

``` console
$ rustup target add thumbv7em-none-eabihf
```

```

**Review by Urhengulas — CHANGES_REQUESTED** [review]

**Comment by Urhengulas** [comment]
Oh man. It seems that the GitHub suggestion feature can't handle "```" :/

@afajl Can you please re-add the closing backticks of the command block and only one newline?

**Comment by afajl** [comment]
:) Sure

**Comment by jonas-schievink** [comment]
Thanks!

bors r+

**Comment by japaric** [comment]
(no CI or bors so I merged manually)

**Pull request merged**